src/sdg/templates/organisaties/_include/product_catalog.html

```diff
@@ -31,7 +31,7 @@ <h4>{{ catalog.naam }}</h4>
                         {% product_icon product %}
                     </div>
                     <div>
-                        <div class="products__item-title">{{ product.upn_label }}</div>
+                        <div class="products__item-title">{{ product.name }}</div>
```

Comment:
Please use a similar construction as done https://github.com/maykinmedia/sdg-invoervoorziening/pull/253/files#diff-afbcf6c0a991e8802d5b7eb70227b5cb12d402dac45ef47ad635b5547271a9d7R213

Add a property `name`, annotate with `_name` so you are sure you can always use `Product.name` and it will just work (annotated or not).